docs/source/en/glossary.mdx

```diff
@@ -237,22 +267,30 @@ A task that combines texts with another kind of inputs (for instance images).
 
 ## N
 
-### NLG
+### Natural language generation
 
-Natural language generation, all tasks related to generating text (for instance talk with transformers, translation).
+All tasks related to generating text (for instance talk with transformers, translation).
 
-### NLP
+### Natural language processing
 
-Natural language processing, a generic way to say "deal with texts".
+A generic way to say "deal with texts".
 
-### NLU
+### Natural language understanding
 
-Natural language understanding, all tasks related to understanding what is in a text (for instance classifying the
+All tasks related to understanding what is in a text (for instance classifying the
 whole text, individual words).
 
 ## P
 
-### Position IDs
+### pixel values
+
+A tensor of the numerical representations of an image that is passed to a model. The pixel values have a shape of [`batch_size`, `num_channels`, `height`, `width`], and are generated from a feature extractor.
+
+### pooling
+
+An operation that reduces a matrix into a smaller matrix, either by taking the maximum or average of the pooled area. Pooling is similar to [convolution](#convolution) in the sense that it is repeated over the entire matrix, and each operation is applied to a different segment of the matrix.
```

Comment:
The second sentence is very confusing. Pooling is very different from convolution, which is applied on windows of the matrix, whereas pooling is applied on one (or several) dims. 
```suggestion
An operation that reduces a matrix into a smaller matrix, either by taking the maximum or average of the pooled dimension(s).
```